pkg/monitor/cluster/clustersync.go

```diff
@@ -0,0 +1,92 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitClusterSync(ctx context.Context) error {
+	clusterSync, err := mon.hiveClusterManager.GetSyncSetResources(ctx, mon.doc)
+	if err != nil {
+		return err
+	}
+
+	totalClusterSyncSuccess, totalClusterSyncFailure := 0, 0
+
+	if clusterSync != nil {
+		if clusterSync.Status.SyncSets != nil {
+			for _, s := range clusterSync.Status.SyncSets {
+				if s.Result == "Success" {
```

Comment:
nit; instead of using "SUCCESS"/"FAILURE" maybe use constants SuccessSyncSetResult and FailureSyncSetResult from here: https://github.com/openshift/hive/blob/9038045312ff411db9cd0e0daa55bbc5752f341d/apis/hiveinternal/v1alpha1/clustersync_types.go#L110C2-L110C22

Reply:
Done